lib/breathe_client.rb

```diff
@@ -43,6 +43,11 @@ def absence_events(person:, after:)
           half_day_at_start = absence.half_start
           half_day_at_end = absence.half_end
 
+          if start_date < Date.parse(after)
```

Comment:
Not for this PR, but we should definately refactor these two chunks (absence/sickness) so that they're a single common function

Reply:
I think this should possibly be end_date, so that if I'm marked as already halfway through an absence, it still gets added to productive?

Theoretically there's a similar scenario for "I've just come back and my line manager has only just marked the sickness in breathe" but that probably shouldn't be happening.

I think this is probably working out the needs the client has :)